src/ansys/motorcad/core/rpc_client_core.py

```diff
@@ -238,6 +242,11 @@ def _resolve_localhost(self):
             SERVER_IP = LOCALHOST_ADDRESS
 
     def _check_address_for_response(self, address):
+        """Try resolving localhost to a specified url.
+
+        We can use a handshake method with Motor-CAD instance to check if connection is successful.
+        """
+        # Need to use SERVER_IP global var to preserve backwards compatibility
```

Comment:
I'm not sure this really explains things clearly enough. Backwards compatibility with what? I thought we needed to use it as _wait_for_response communicates using the address from get_url, which in turn uses the global SERVER_IP

Reply:
Maybe not super clear comment. I'd like to get rid of SERVER_IP ideally and have this as a class var. I've used it in a few places however so I'm going to keep it. I'm hoping url parameter will make this obsolete anyway. I'll improve the comments

Reply:
Looks good, cheers